api/paging.go

```diff
@@ -26,21 +26,16 @@ type Paging struct {
 	after domain.After
 }
 
-// defaultPagingOptions returns default paging options: offset 0, limit 20, default sorting, ascending
+// defaultPagingOptions returns default paging options: offset 0, limit 0 (not applied), default sorting, ascending
 func defaultPaging() *Paging {
-	return &Paging{limit: 20, offset: 0, sortBy: "", descending: false, after: ""}
+	return &Paging{limit: 0, offset: 0, sortBy: "", descending: false, after: ""}
 }
 
 // PagingWithLimit sets limit option - maximum number of items returned.
-// Default 20, minimum 1 and maximum 100.
+// For buckets and organization the  default 20, minimum 1 and maximum 100,
```

Comment:
Is the default controlled by InfluxDB server. Wouldn't be better to express the defaults in buckets/orgs APIs.

Reply:
Ok. Orgs and Bucket API already have this comment